### PR TITLE
Fix #210: Remove summary of mitigations and include requirements in m…

### LINKED ISSUE #210: Information Stored on User Devices Mitigations section is structurally inconsistent.
PR #202 fixes the requirement inconsistencies identified in issue #187. However, the structure of the Mitigations subsection of the [Information Stored on User Devices](https://w3c.github.io/encrypted-media/#privacy-storedinfo) section remains internally inconsistent and inconsistent with other Mitigations sections. This issue tracks editorial changes to address this and is low priority.

Specifically:
* The Mitigations section begins with "In summary," but not all items in the summary are covered in the following subsections.
* This is the only Mitigations section that includes a "summary." All others either do not have subsections or only have subsections.

### PR DESCRIPTION
**Pull request by mwatson2**
…ain text

### CODE REVIEW

**Comment by ddorwin** [comment]
I'll review this tomorrow morning. @mwatson2, do you want to resolve the conflicts before I do so?

**Comment by mwatson2** [comment]
Done

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -3969,33 +3969,29 @@
...
-            <li><p>MUST ensure it is restricted to the <a def-id="origin"></a> for which it was created.</p></li>
-            <li><p>MUST ensure it is restricted to the current <a def-id="browsing-profile"></a> and does not leak to or from private browsing sessions.</p></li>
-            <li><p>MUST allow the user to clear it.</p></li>
-            <li><p>SHOULD allow the user to clear it by <a def-id="origin"></a>.</p></li>
-            <li><p>SHOULD treat it like other site data, including presenting it along with cookies [[!COOKIES]], including it in "remove all data", and presenting it in the same UI locations.</p></li>
-          </ul>
-
           <dl>
-            <dt>Origin-specific Key System storage</dt>
+            <dt>Use Origin-specific and user agent profile-specific Key System storage</dt>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: "Use" is inconsistent with other headers in this section but consistent with those in other sections. I think this is probably better. I'll make them all consistent after this lands.

File: encrypted-media-respec.html [hunk 2]
```diff
@@ -3969,33 +3969,29 @@
...
             <dd>
-              <p>Any data persistently stored by the CDM that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific way.
-                Session data, licenses, and keys that are persistently stored MUST be stored per-<a def-id="origin"></a>.
+              <p>Any data persistently stored by the CDM that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific and user agent profile-specific way and MUST NOT leak to or from private browsing sessions.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: `s/user agent profile/<a def-id="browsing-profile"></a>/`

File: encrypted-media-respec.html [hunk 3]
```diff
@@ -3969,33 +3969,29 @@
...
+                Session data, licenses, and keys that are persistently stored MUST be stored per-<a def-id="origin"></a> and per user agent profile.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: ditto

File: encrypted-media-respec.html [hunk 4]
```diff
@@ -3969,33 +3969,29 @@
...
+                User Agents MUST allow the user to clear data persisted by the CDM and SHOULD allow users to clear that data by <a def-id="origin"></a>.
+                User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> and/or for all origins.
                 User agents SHOULD help the user avoid <a href="#incomplete-clearing">Incomplete Clearing of Data</a>.
               </p>
             </dd>
 
             <dt>Treating <a def-id="distinctive-identifiers"></a> and Key System stored data like cookies / web storage</dt>
             <dd>
+              <p>
+                User Agents SHOULD treat data persisted by CDMs like other site data, including presenting it along with cookies
```
Comment (ddorwin) [inline encrypted-media-respec.html]: We might want to integrate some of these details into the #like-cookies section referenced below.

**Comment by ddorwin** [comment]
Thanks. LGTM with comments.

**Pull request merged**